.circleci/config.yml

```diff
@@ -8,7 +8,11 @@ jobs:
       - checkout
       - restore_cache:
           key: v1-deps-{{ checksum "pom.xml" }}
-      - run: mvn -B dependency:go-offline compile test-compile
+      - run:
+          environment:
+            # Maven heap size
+            MAVEN_OPTS: -Xmx256m -Xms256m
```

Comment:
⭐️

Reply:
Still not working. Will check later.

Reply:
`[ERROR] GC overhead limit exceeded -> [Help 1]` not enough now